From 750fbe7bdc18fb5e7ffa65dabd5aa0596d4727cb Mon Sep 17 00:00:00 2001
From: make shi <b15407@freescale.com>
Date: Wed, 29 Aug 2012 17:53:38 +0800
Subject: [PATCH 1984/2463] ENGR00221716-02 Mx6 USB host: add port speed
 define MACRO to arc_otg.h

Add port speed define MACRO to arc_otg.h.

Signed-off-by: make shi <b15407@freescale.com>
---
 arch/arm/plat-mxc/include/mach/arc_otg.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/arc_otg.h b/arch/arm/plat-mxc/include/mach/arc_otg.h
index bdb6b25..f2bb01c 100755
--- a/arch/arm/plat-mxc/include/mach/arc_otg.h
+++ b/arch/arm/plat-mxc/include/mach/arc_otg.h
@@ -246,6 +246,11 @@ extern void __iomem *imx_otg_base;
 #define PORTSC_PTS_SERIAL		(3 << 30)	/* serial */
 #define PORTSC_STS			(1 << 29)	/* serial xcvr select */
 #define PORTSC_PTW                      (1 << 28)       /* UTMI width */
+#define PORTSC_PORT_SPEED_FULL		(0x0<<26)
+#define PORTSC_PORT_SPEED_LOW		       (0x1<<26)
+#define PORTSC_PORT_SPEED_HIGH		(0x2<<26)
+#define PORTSC_PORT_SPEED_UNDEF		(0x3<<26)
+#define PORTSC_PORT_SPEED_MASK		(0x3<<26)
 #define PORTSC_HSIC_MODE                (1 << 25)       /* Only for HSIC */
 #define PORTSC_PHCD                     (1 << 23)       /* Low Power Suspend */
 #define PORTSC_WKDC			(1 << 21)	/* wakeup on discnt*/
-- 
1.7.7.4

